api/cluster/pdb.go

```diff
@@ -113,11 +125,15 @@ func (c *controller) deployPodDisruptionBudget(ctx context.Context, pdb *PodDisr
 		return err
 	}
 
-	pdbCfg := policyv1cfg.PodDisruptionBudget(pdb.Name, pdb.Namespace)
-	pdbCfg.WithLabels(pdb.Labels)
-	pdbCfg.WithSpec(pdbSpec)
+	pdbJSON, err := json.Marshal(pdbSpec)
+	if err != nil {
+		return err
+	}
+
+	forceEnabled := true
 
-	_, err = c.policyClient.PodDisruptionBudgets(pdb.Namespace).Apply(ctx, pdbCfg, metav1.ApplyOptions{FieldManager: "application/apply-patch"})
+	_, err = c.policyClient.PodDisruptionBudgets(pdb.Namespace).
+		Patch(ctx, pdb.Name, types.ApplyPatchType, pdbJSON, metav1.PatchOptions{FieldManager: "application/apply-patch", Force: &forceEnabled})
```

Comment:
why do we force patch instead of just apply, i thought apply would do the same? I might be missing something here